terraform/aks/05-mis/02-inputs-default.tf

```
@@ -46,6 +46,30 @@ locals {
     "environment"        = local.env_display_names[var.environment]
     "criticality"        = local.criticality[var.environment]
   }
+
+  log_analytics_env_mapping = {
+    sandbox = ["sbox"]
+    nonprod = ["dev", "test", "ithc", "demo", "stg"]
+    prod    = ["prod", "mgmt"]
+  }
+
+  log_analytics_workspace = {
+    sandbox = {
+      subscription_id = "bf308a5c-0624-4334-8ff8-8dca9fd43783"
+      name            = "hmcts-sandbox"
+    }
+    nonprod = {
+      subscription_id = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
+      name            = "hmcts-nonprod"
+    }
+    prod = {
+      subscription_id = "8999dec3-0104-4a27-94ee-6588559729d1"
+      name            = "hmcts-prod"
+    }
+  }
+  resolved_subscription_id = local.log_analytics_workspace[[for x in keys(local.log_analytics_env_mapping) : x if contains(local.log_analytics_env_mapping[x], var.environment)][0]].subscription_id
```

this should have a better name

```suggestion
  log_analytics_subscription_id = local.log_analytics_workspace[[for x in keys(local.log_analytics_env_mapping) : x if contains(local.log_analytics_env_mapping[x], var.environment)][0]].subscription_id
```